irmin.opam

```diff
@@ -35,6 +35,7 @@ depends: [
   "vector" {with-test}
   "mdx" {>= "2.0.0" & with-test}
   "odoc" {(< "2.0.1" | > "2.0.2") & with-doc} # See https://github.com/ocaml/odoc/issues/793
+  "bisect_ppx" {dev & >= "2.5.0"}
```

Comment:
Shouldn't this be added with the `with-test` flag?

Reply:
Strictly speaking, the `with-test` flag guards the [corresponding `with-test` action](https://github.com/mirage/irmin/blob/36056b27b119a6b6774528e3805cf6f572d679e3/irmin.opam#L13) in the `opam` file, which doesn't run with instrumentation. We can add it anyway, but this means adding it in every `opam` file we own as `with-test` is not transitive. WDYT?

Reply:
You are right, I haven't thought of this! You can ignore this comment and leave it this way.